source/conf.ts

```diff
@@ -272,190 +516,16 @@ class Conf {
-		const {store} = this;
-		if (this._options.accessPropertiesByDotNotation) {
-			dotProp.delete(store, key);
-		} else {
-			delete store[key];
-		}
-
-		this.store = store;
-	}
-
-	clear() {
-		this.store = plainObject();
-	}
-
-	onDidChange(key, callback) {
-		if (typeof key !== 'string') {
-			throw new TypeError(`Expected \`key\` to be of type \`string\`, got ${typeof key}`);
-		}
-
-		if (typeof callback !== 'function') {
-			throw new TypeError(`Expected \`callback\` to be of type \`function\`, got ${typeof callback}`);
-		}
-
-		const getter = () => this.get(key);
-
-		return this.handleChange(getter, callback);
-	}
-
-	onDidAnyChange(callback) {
-		if (typeof callback !== 'function') {
-			throw new TypeError(`Expected \`callback\` to be of type \`function\`, got ${typeof callback}`);
-		}
-
-		const getter = () => this.store;
-
-		return this.handleChange(getter, callback);
-	}
-
-	handleChange(getter, callback) {
-		let currentValue = getter();
-
-		const onChange = () => {
-			const oldValue = currentValue;
-			const newValue = getter();
-
-			try {
-				// TODO: Use `util.isDeepStrictEqual` when targeting Node.js 10
-				assert.deepEqual(newValue, oldValue);
-			} catch (_) {
-				currentValue = newValue;
-				callback.call(this, newValue, oldValue);
-			}
-		};
-
-		this.events.on('change', onChange);
-		return () => this.events.removeListener('change', onChange);
-	}
-
-	get size() {
-		return Object.keys(this.store).length;
-	}
-
-	get store() {
-		try {
-			let data = fs.readFileSync(this.path, this.encryptionKey ? null : 'utf8');
-
-			if (this.encryptionKey) {
-				try {
-					// Check if an initialization vector has been used to encrypt the data
-					if (data.slice(16, 17).toString() === ':') {
-						const initializationVector = data.slice(0, 16);
-						const password = crypto.pbkdf2Sync(this.encryptionKey, initializationVector.toString(), 10000, 32, 'sha512');
-						const decipher = crypto.createDecipheriv(encryptionAlgorithm, password, initializationVector);
-						data = Buffer.concat([decipher.update(data.slice(17)), decipher.final()]);
-					} else {
-						const decipher = crypto.createDecipher(encryptionAlgorithm, this.encryptionKey);
-						data = Buffer.concat([decipher.update(data), decipher.final()]);
-					}
-				} catch (_) {}
-			}
-
-			data = this.deserialize(data);
-			this._validate(data);
-			return Object.assign(plainObject(), data);
-		} catch (error) {
-			if (error.code === 'ENOENT') {
-				this._ensureDirectory();
-				return plainObject();
-			}
-
-			if (this._options.clearInvalidConfig && error.name === 'SyntaxError') {
-				return plainObject();
-			}
-
-			throw error;
-		}
-	}
-
-	set store(value) {
-		this._ensureDirectory();
-
-		this._validate(value);
-		this._write(value);
-
-		this.events.emit('change');
-	}
-
-	* [Symbol.iterator]() {
-		for (const [key, value] of Object.entries(this.store)) {
-			yield [key, value];
-		}
-	}
 }
 
-module.exports = Conf;
+export default Conf;
```

Comment:
Can you add this hack: https://github.com/sindresorhus/is/blob/05cdaccf924c37b54f4fdd149caab5ebe2af2f1b/source/index.ts#L655-L657 So CommonJS projects can import this without having to access Conf with `conf.default`.